src/tests/integration/test_generic_db_opeations.py

```diff
@@ -144,3 +144,13 @@ def test_telemetry_execute_command_has_connection_info(
             },
         },
     )
+
+
+def test_sql_cmd_magic_uno(ip):
+    result = ip.run_cell(
+        "%sqlcmd test --table author --column year_of_death"
+        " --less-than 1700 --greater 1600"
+    ).result
+
+    assert len(result) == 2
+    assert "William" in str(result["less_than"])
```

Comment:
Good job moving to integration testing.
Since the philosophy of testing is to cover every part of code we wrote, let's put more similar test inputs to the test `greater_or_equal` and `less_than_or_equal`

You can also use @pytest.mark.parametrize to organize your multiple test inputs.

Something like:
```python
@pytest.mark.parametrize(
    "cell, expected",
    [
        (
            "%sqlcmd test --table author --column year_of_death --less-than 1700 --greater 1600",
            2,
        ),
        (
            "%sqlcmd test --table author --column year_of_death --greater 1600",
            2,
        ),
        ...
    ],
)
def test_sql_cmd_magic_uno(ip, cell, expected):
  result = ip.run_cell(cell)
  assert len(result) == expected
```